producer-kafka/src/main/java/io/stargate/producer/kafka/configuration/DefaultConfigLoader.java

```diff
@@ -29,22 +30,32 @@
 import org.apache.kafka.clients.producer.ProducerConfig;
 
 public class DefaultConfigLoader implements ConfigLoader {
-
+  public static final String CONFIG_STORE_MODULE_NAME = "cdc.kafka";
   private static final Pattern CDC_KAFKA_PRODUCER_SETTING_PATTERN =
       Pattern.compile(String.format("^(%s)\\.(.*.)", CDC_KAFKA_PRODUCER_SETTING_PREFIX));
 
   @Override
-  public CDCKafkaConfig loadConfig(Map<String, Object> options) {
+  public CDCKafkaConfig loadConfig(ConfigStore configStore) {
+    return loadConfig(configStore.getConfigForModule(CONFIG_STORE_MODULE_NAME).getConfigMap());
```

Comment:
Sorry, I'm a bit lost... Do config overrides happen in this case?

Reply:
When I was thinking about it more, I think that currently, we don't need to support them in the CDC producer - it is documented in the `README.md.` The main reason behind this is that customers can provide any Kafka producer setting using `cdc.kafka.producer.` Because of that, we need to operate on the Map directly.
I think that there is no need for supporting overrides here. 
The override mechanism for config-store was created mainly for this:
https://github.com/stargate/stargate/issues/332

Reply:
I see. Thanks!